### DISCUSSION TITLE
Live examples or description of field value

### DISCUSSION THREAD

**Discussion by umangdadhich** [CMSgov/price-transparency-guide#140] [Q&A] [answered]
I am looking for some live examples of how group health plans are extracted for the field value(reporting_entity_name, reporting_entity_type). Any source or someone has created it?

**Comment by shaselton-usds** [comment, marked as answer]
@umangdadhich I'm not quite sure what is being asked here -- is it related to this discussion https://github.com/CMSgov/price-transparency-guide/discussions/36?
